app/models/form526_submission.rb

```diff
@@ -129,6 +129,11 @@ class Form526Submission < ApplicationRecord
 
   validates(:auth_headers_json, presence: true)
 
+  scope :pending_backup_submissions, lambda {
+    where(aasm_state: 'delivered_to_backup')
+      .where.not(backup_submitted_claim_id: nil)
```

Comment:
Wouldn't `pending_backup_submissions` have a `backup_submitted_claim_id` that _is_ nil?

Reply:
the state `delivered_to_backup` means that it's been sent (successfully) to LH.  HOWEVER, LH may still reject it later on, after their system has reviewed it.  This is definitely confusing, and is also the central reason we need polling.  SO the above code is really just saying "give me anything with a backup_submitted_claim_id that hasn't been finalized"

Reply:
I could call it `delivered_to_backup_pending_approval`?

Reply:
Got it. I suppose I wasn't clear on what the state of `delivered_to_backup` implied. It makes sense now that you've explained it and I referenced your earlier PR. If "backup" is LH, and the `backup_submitted _claim_id` is a unique ID received from LH acknowledging receipt of a backup submitted claim, then the given name would work. Or, if not, and wanting to keep it concise, a name like `pending_backup_finalization`, perhaps. I'll defer to you though as I feel you're better with words.

Reply:
I think we are going to get an itteration on these states in the near future anyway for two reasons
- We need to add two more for remediation stuff
- these aren't really "states" as much as they are "end of life tags" or "remediation notes"
I'm ok with leaving it for now; but for the record [my next TODO has an action item to clean up / document what these states all mean](https://app.zenhub.com/workspaces/disability-benefits-experience-team-carbs-6470c8bfffee9809b2634a52/issues/gh/department-of-veterans-affairs/va.gov-team/80624)